bin/pulsar

```diff
@@ -48,6 +52,8 @@ PRESTO_HOME=${PULSAR_HOME}/lib/presto
 # Check bookkeeper env and load bkenv.sh
 if [ -f "$PULSAR_HOME/conf/bkenv.sh" ]
 then
+    BOOKIE_LOG_DIR=${BOOKIE_LOG_DIR:-"${PULSAR_LOG_DIR}"}
+    BOOKIE_GC_LOG_DIR=${BOOKIE_GC_LOG_DIR:-"${PULSAR_GC_LOG_DIR}"}
```

Comment:
And I suggest we assign  `BOOKIE_GC_LOG_DIR`  and `PULSAR_LOG_DIR` in bk_env.sh and pulsar_env.sh. Looks more straightforward.

Reply:
Addressed in latest commit.